### PR TITLE
LPS-163032 number available can't be less than 0

### PR DESCRIPTION
**Pull request by amosfong**
Hey guys, I barely know react, so review thoroughly!

### CODE REVIEW

**Comment by amosfong** [comment]
ci:test:sf

**Comment by amosfong** [comment]
ci:test:relevant

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/containers/setup-forms/InviteTeamMembersForm/index.js [hunk 1]
```diff
@@ -387,7 +387,11 @@ const InviteTeamMembersPage = ({
 												  )
 										}
 										  ${i18n.sub('available-x-of-x', [
-												availableAdminsRoles,
+												`${
```
Comment (claricebrasil) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/containers/setup-forms/InviteTeamMembersForm/index.js:390]: Hi @amosfong thanks for the help! In this case you don't need to use string template, you can just use:
```suggestion
												availableAdminsRoles < 0 ? 0 : availableAdminsRoles
```

**Comment by claricebrasil** [comment]
Resent #2188

**Comment by amosfong** [comment]
@claricebrasil ok thanks! I copied from here https://github.com/liferay/liferay-portal/blob/5695541b156de63333724e09899afd59ec0f2fa8/modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/pages/Project/TeamMembers/components/TeamMembersTable/components/TeamMembersTableHeader/TeamMembersTableHeader.js#L101

Should they be the same or this is a different use case?

**Pull request closed without merging**